### PR TITLE
ESLint entire source directory

### PR DESCRIPTION
**Pull request by OliverJAsh**
- moves the config away from Grunt so it can be shared with IDE plugins.
- lints all JS files in `static/src`, including bootstraps (not previously linted)

### CODE REVIEW

**Comment by johnduffell** [comment]
good good! :+1: no idea why they weren't linted before

File: grunt-configs/eslint.js [hunk 1]
```diff
@@ -4,71 +4,19 @@ module.exports = function(grunt, options) {
...
-        facia: {
-            files: [{
-                expand: true,
-                cwd: 'static/src/javascripts/projects/facia',
-                src: [
-                    '**/*.js'
-                ]
-            }]
-        },
-        'facia-tool': {
```
Comment (piuccio) [inline grunt-configs/eslint.js]: Did you just kill linting for facia tool?

Comment (OliverJAsh) [inline grunt-configs/eslint.js]: No, if you look at the linting pattern above…

On Tue, 23 Jun 2015 at 11:43 Fabio Crisci <notifications@github.com> wrote:

> In grunt-configs/eslint.js
> <https://github.com/guardian/frontend/pull/9578#discussion_r33028817>:
>
> > -                src: [
> > -                    '**/*.js',
> > -                    '!utils/atob.js'
> > -                ]
> > -            }]
> > -        },
> > -        facia: {
> > -            files: [{
> > -                expand: true,
> > -                cwd: 'static/src/javascripts/projects/facia',
> > -                src: [
> > -                    '**/*.js'
> > -                ]
> > -            }]
> > -        },
> > -        'facia-tool': {
>
> Did you just kill linting for facia tool?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/9578/files#r33028817>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

Comment (piuccio) [inline grunt-configs/eslint.js]: Sory, do you mean this?

```
        all: {
            files: [{
                expand: true,
                cwd: 'static/src',
                src: [
                    '**/*.js'
                ]
            }],
```

That doesn't match facia-tool `cwd: 'facia-tool/public/js/',`

Comment (OliverJAsh) [inline grunt-configs/eslint.js]: Ah, correct, will fix now.

**Pull request merged**